app/views/layouts/email.html.erb

```diff
@@ -1,13 +1,13 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <meta charset="utf-8">
     <title>C2</title>
-    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+    <meta name="viewport" content="width=device-width" />
     <link rel="stylesheet" type="text/css" href="<%= stylesheet_url("mailer_new.css") %>" />
+    <%= render 'mail_shared/outlook_fix' %>
   </head>
   <body>
     <%= yield %>
-    <%= render partial: "mail_shared/footer" %>
```

Comment:
why not put the footer code into a shared partial? I think we will be using it for all emails so good to just have in layout. can take lines 142 -> 157 from attachment template and put it in `mail_shared/_footer.html.haml`